### PR TITLE
Added checkstyle rules for version 8.29

### PR DESCRIPTION
**Pull request by seiferma**
This is just a copy of the existing rules for version 8.27 that we have to create when merging PalladioSimulator/Palladio-Build-MavenParent#28.

### CODE REVIEW

File: misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+<module name="Checker">
+  <property name="severity" value="warning"/>
+  <module name="LineLength">
+    <property name="severity" value="info"/>
+    <property name="ignorePattern" value="\*|import"/>
+    <property name="max" value="130"/>
+  </module>
+  <module name="TreeWalker">
+    <property name="tabWidth" value="4"/>
+    <module name="JavadocMethod">
```
Comment (majuwa) [inline misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml]: I think the properties of this check were removed in 8:28  see [Github issue](https://github.com/checkstyle/checkstyle/issues/7329)
Therefore we also should remove them

File: misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      <property name="allowUndeclaredRTE" value="true"/>
+      <property name="suppressLoadErrors" value="true"/>
```
Comment (majuwa) [inline misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml]: I think the properties of this check were removed in 8:28 see Github issue
Therefore we also should remove them

**Review by majuwa — CHANGES_REQUESTED** [review]
We should remove the properties of the JavadocMethod module since it seems, that they have been removed since 8.28. [Github Issue](https://github.com/checkstyle/checkstyle/issues/7329)

File: misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml (hunk 2 above)
Comment (seiferma) [reply misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml]: You are right. I removed the whole Javadoc section.

**Review by majuwa — APPROVED** [review]
Works for me

**Pull request merged**